src/main/kotlin/no/nav/familie/ks/sak/bisys/BisysService.kt

```diff
@@ -44,12 +44,13 @@ class BisysService(
 
         // hent utbetalingsinfo from infotrygd
         val respons = infotrygdReplikaClient.hentKontantstøttePerioderFraInfotrygd(barnIdenter)
+        logger.info("Hentet ${respons.data.size} data fra infotrygd")
         val utbetalingsinfoFraInfotrygd = respons.data.map { stonad ->
             val barn = stonad.barn.first { stonadBarn -> barnIdenter.any { it == stonadBarn.fnr.asString } }.fnr.asString
             barn to UtbetalingsinfoDto(
-                fomMåned = checkNotNull(stonad.fom),
-                tomMåned = checkNotNull(stonad.tom),
-                beløp = checkNotNull(stonad.belop)
+                fomMåned = checkNotNull(stonad.fom) { "fom kan ikke være null" },
```

Comment:
```suggestion
                fomMåned = checkNotNull(stonad.fom) { "fomMåned kan ikke være null" },
```
1-1 med feltnavn i dto